folium/plugins/realtime.py

```diff
@@ -109,7 +109,7 @@ def __init__(
         get_feature_id: Union[JsCode, str, None] = None,
         update_feature: Union[JsCode, str, None] = None,
         remove_missing: bool = False,
-        container: Optional[Layer] = None,
+        container: Optional[Union[FeatureGroup, GeoJson]] = None,
```

Comment:
this should also be updated in the docstring

Reply:
Done